src/main/java/com/hotels/bdp/circustrain/bigquery/extraction/container/DeleteTableAction.java

```diff
@@ -31,8 +32,12 @@ public DeleteTableAction(Table table) {
 
   @Override
   public void run() {
-    table.delete();
-    log.debug("Deleted table '{}'", table.getTableId());
+    try {
+      table.delete();
+      log.debug("Deleted table '{}'", table.getTableId());
+    } catch (BigQueryException e) {
+      log.error("Could not delete BigQuery table '{}': {}", table.getTableId(), e.getMessage());
```

Comment:
We should log `e` to get the full stack trace, not just `e.getMessage()`.

Reply:
Does it mean it would turn into 
```
log.error("Could not delete BigQuery table '{}': ", table.getTableId(), e);
```
?

Reply:
`log.error("Could not delete BigQuery table '{}'", table.getTableId(), e);`

No need for the ":" as the exception is already logged separately by the framework. You can try it out yourself in a simple main method to see the different outputs:

```
  public static void main(String[] args) {
    try {
      throw new RuntimeException("bla");
    } catch (Exception e) {
    log.error("Could not delete BigQuery table '{}'", "mytable", e);}
  }
```

Reply:
Ah okay, I realised that I had a tests that showed how the error is displayed. This is nicer, thank you.